src/libraries/icubmod/dragonfly2/common/DragonflyDeviceDriver2.h

```diff
@@ -92,22 +92,22 @@ The video_type parameter determines how images are acquired by the dragonfly chi
 
 \subsection port_units Port, Unit number and 64 bit Global Unique Identifier
 
-Many cameras can coexist on the same Firewire bus (port), sharing the available bandwidth. Each camera connected to the same Firewire bus is associated to a unit number. Port numbers, as well as unit numbers, are assigned increasingly starting from 0. 
+Many cameras can coexist on the same Firewire bus (port), sharing the available bandwidth. Each camera connected to the same Firewire bus is associated to a unit number. Port numbers, as well as unit numbers, are assigned increasingly starting from 0.
 The iCub robot has one only Firewire card, and thus its port number is always 0 (can be omitted). The two left and right cameras will be unit 0 and 1, but unfortunately this association is not deterministic, and thus the Global Unique Identifier (guid) must be used in order to univokely identify a camera.
-The icubmoddev device driver supports configuration files, thus the most convenient way to deal with GUIDs is putting them in .ini files that will be passed to icubmoddev together with all the other parameters as follows:
+The yarpdev device driver supports configuration files, thus the most convenient way to deal with GUIDs is putting them in .ini files that will be passed to yarpdev together with all the other parameters as follows:
 
-icubmoddev --from camera/dragonfly2_config_left.ini
-icubmoddev --from camera/dragonfly2_config_right.ini 
+yarpdev --from camera/dragonfly2_config_left.ini
+yarpdev --from camera/dragonfly2_config_right.ini
 
-In the latest iCub software releases the .ini files are already supplied in <B>app/robots/iCubXXXnn/camera</B> folders. The --d option must be replaced in them by assigning the guid of the corresponding camera to the guid parameter. 
+In the latest iCub software releases the .ini files are already supplied in <B>app/robots/iCubXXXnn/camera</B> folders. The --d option must be replaced in them by assigning the guid of the corresponding camera to the guid parameter.
```

Comment:
I think that the following is not true anymore: "In the latest iCub software releases the .ini files are already supplied in app/robots/iCubXXXnn/camera folders". Perhaps point to the robots-configuration repository? Or to the robot init files installed in .local with the yarp-config directory.

Reply:
I agree, but this was just a cleanup commit and I am quite ignorant about all the things related to the camera configuration files. I think we can open a separate issue for this.

Reply:
Separate issue opened in https://github.com/robotology/icub-main/issues/512 .